<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgorithmDialogClass</class>
 <widget class="QMainWindow" name="AlgorithmDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlgorithmDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AlgorithmDialogClass{
background-image: url(:/image/r1.jpg);
border-radius:5px;
border:1px groove gray;
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>输入函数</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border:1px groove gray</string>
        </property>
        <property name="maxLength">
         <number>65536</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>一阶导数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border:1px groove gray;
border-style: outset;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>二阶导数</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QToolButton" name="btnCalculate">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
color:blue;
border-radius:5px;
border:1px groove gray;
border-style: outset
}</string>
        </property>
        <property name="text">
         <string>计算</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QToolButton" name="buttonClear">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
color:blue;
border-radius:5px;
border:1px groove gray;
border-style:outset

}</string>
        </property>
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border:1px groove gray;
border-style: outset;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelLeft">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>输入左端点</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineTextLeft">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border:1px groove gray;
border-style: outset;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelRight">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>输入右端点</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineTextRight">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border:1px groove gray;
border-style: outset;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelAcc">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>精度</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineTextAccuracy">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border:1px groove gray;
border-style: outset;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelInitPoint">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>随机初始点</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditInitPoint">
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border:1px groove gray;
border-style: outset;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
